custom-aa/test/utils/utils.ts

```diff
@@ -0,0 +1,165 @@
+import * as hre from "hardhat";
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+
+import { EIP712Signer, Provider, types, utils, Wallet } from "zksync-web3";
+import { localConfig } from "../../../tests/testConfig";
+import * as ethers from "ethers";
+
+export const deployFactory = async (
+  privateKey: string = localConfig.privateKey,
+) => {
+  // Private key of the account used to deploy
+  const wallet = new Wallet(privateKey);
+  const deployer = new Deployer(hre, wallet);
+  const factoryArtifact = await deployer.loadArtifact("AAFactory");
+  const aaArtifact = await deployer.loadArtifact("TwoUserMultisig");
+
+  // Getting the bytecodeHash of the account
+  const bytecodeHash = utils.hashBytecode(aaArtifact.bytecode);
+
+  const factory = await deployer.deploy(
+    factoryArtifact,
+    [bytecodeHash],
+    undefined,
+    [
+      // Since the factory requires the code of the multisig to be available,
+      // we should pass it here as well.
+      aaArtifact.bytecode,
+    ],
+  );
+
+  console.log(`AA factory address: ${factory.address}`);
+
+  return factory;
+};
+
+export const deployMultisig = async (factoryAddress: string) => {
```

Comment:
The name here is slightly misleading as this function does three things: 
1. deploys the multisig
2. sends funds to the multisig
3. multisig performs a transaction